portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -38,12 +41,26 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
 
+    String strNumComments = request.getParameter("numComments");
+    int numComments;
+    try{ 
+      numComments = Integer.parseInt(strNumComments);
+      if(numComments < 0) {
```

Comment:
Should I be returning an error message if 0 comments are requested? Because technically, it is valid to ask for 0 comments in the query.

Reply:
In general, I would say it is better to return an error if your program doesn't support an operation because it doesn't make sense, even if it can technically support it. Any feedback to tell the API user why there are no results is helpful.